conda/pygpu/meta.yaml

```diff
@@ -19,14 +18,14 @@ requirements:
     - git            [not win]
     - python
     - cython >=0.25
-    - numpy x.x
+    - numpy 1.11
```

Comment:
Does it mean it will only support numpy 1.11? Why not bump that to 1.13 to support the more recent version?

Reply:
No, it was discovered that numpy is forward-compatible.  So it should work with numpy 1.12 and 1.13 too (and 1.14 whenever it comes out).